platform-api/src/main/java/com/flow/platform/api/domain/Job.java

```diff
@@ -69,7 +71,24 @@
     @Expose
     private Integer number;
 
```

Comment:
@Expose
private NodeResult result; // for root node result which the path equal to job.nodePath